src/styles/partials/components/_button.scss

```diff
@@ -1,108 +1,112 @@
 .dg_button,
 .dg_button-secondary {
-	&:hover,
-	&.hovered {
-		background: $vibrant-blue;
-	}
-
-	&:active,
-	&.active {
-		background: $pale-blue;
-	}
-
-	&:disabled,
-	&.disabled {
-		border-color: $gray-light;
-	}
+  &:hover,
+  &.hovered {
+    background: $vibrant-blue;
+  }
+
+  &:active,
+  &.active {
+    background: $pale-blue;
+  }
+
+  &:disabled,
+  &.disabled {
+    border-color: $gray-light;
+  }
+}
+
+a.dg_button-secondary {
```

Comment:
Wondering if the `text-decroation: none;` property should be nested under `.dg_button, .dg_button-secondary`. Doing this way, it would apply to both button classes should either be tied to an a tag.

Reply:
we could, this only applies to the link, do you think a comment would help out here?

Reply:
If this will only apply to a link using the `dg_button-secondary ` class now and in to the future, then it should be fine as-is without a comment added. If down the road, we would like to use `dg_button` class with a link as well, in that case I would nest.

Reply:
I think for now this is the only use case. Will add a comment.

Reply:
ended up removing text decoration from any state for a secondary button.